bool start
bool publish
string folder_name
----
